src/cascade/input_data/db/mortality.py

```diff
@@ -0,0 +1,76 @@
+import pandas as pd
+
+from cascade.core.db import connection
+
+
+def get_excess_mortality_data(execution_context):
+    model_version_id = execution_context.parameters.model_version_id
+
+    query = """
+     SELECT
+                model_version_dismod_id as data_id,
+                location_id,
+                year_start as time_lower,
+                age_start as age_lower,
+                age_end as age_upper,
+                sex_id as x_sex,
+                mean as meas_value,
+                lower as meas_lower,
+                upper as meas_upper,
+                standard_error
+            FROM
+                epi.t3_model_version_emr t3_emr
+            WHERE model_version_id = %(model_version_id)s
+            AND t3_emr.outlier_type_id = 0
+    """
+
+    with connection(execution_context) as c:
+        data = pd.read_sql(query, c, params={"model_version_id": model_version_id})
+
+    return data
+
+
+def get_cause_specific_mortality_data(execution_context):
+    model_version_id = execution_context.parameters.model_version_id
+
+    query = """
+            SELECT
+                location_id,
+                year_id as time_lower,
+                age_group_id,
+                sex_id as x_sex,
+                mean as meas_value,
+                lower as meas_lower,
+                upper as meas_upper
+            FROM
+                epi.t3_model_version_csmr t3_csmr
```

Comment:
I am wondering why there are 2 items on the FROM line?
epi.t3_model_version_csmr 
t3_csmr

Reply:
They're creating an alias for the table name. If you look at the rest of the query you can see them refer to it by the shorter version.

Reply:
I don't see it being used in lines 36-48 which is to the end of the query, except in line 46.

Reply:
Just saying, I noticed in the file a couple places where the FROM line has the two names, but the shorter name doesn't appear anywhere else in the query.  I did see in at least one query the shorter name is used.  Probably not a big deal.

Reply:
Ah, you're right. It's superfluous here. I must have been looking at one of the other queries.